src/phantom_eval/prompts.py

```diff
@@ -105,12 +105,16 @@ def get_prompt(self, prolog_query: bool = False) -> PromptTemplate:
 Answer: hobby(X, "meteorology"), dob(X, Y)
 
 Example 8:
-Question: Who is the cousin of the person whose occupation is broadcast engineer?
-Answer: job(Y, "broadcast engineer"), cousin(Y, X)
-
-Example 9:
 Question: Who is the granddaughter of the mother of the friend of the friend of the mother of the parent of the friend of the great-granddaughter of the person whose occupation is theatre manager?
 Answer: job(A, "theatre manager"), great_granddaughter(A, B), friend(B, C), parent(C, D), mother(D, E), friend(E, F), friend(F, G), mother(G, H), granddaughter(H, I)
+
+Example 9:
+Question: How many friends does the child of Alvaro Smock have?
+Answer: child(A, 'Alvaro Smock'), findall(B, friend(A, B), C), length(C, D)
+
+Example 10:
+Question: How many uncles does the maternal grandmother of the friend of Stacia Toombs have?
+Answer: friend(A, 'Stacia Toombs'), mother(A, B), mother(B, C), findall(D, uncle(C, D), E), length(E, F)
```

Comment:
```suggestion
Answer: friend("Stacia Toombs", A), mother(A, B), mother(B, C), findall(D, uncle(C, D), E), length(E, F)
```
Similar comment as in Example 9